src/filters/apodization.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 /**
```

Comment:
please remove those header comments. They are added automatically by webstorm (this can be changed)